.github/workflows/github-actions-tests.yml

```
@@ -15,27 +15,29 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [ "3.8", "3.9", "3.10", "3.11", "3.12", "3.13" ]
+        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13", "3.13t" ]
```

Keep it consistent, since a space was removed from the beginning.

```suggestion
        python-version: ["3.9", "3.10", "3.11", "3.12", "3.13", "3.13t"]
```